index.js

```diff
@@ -0,0 +1,19 @@
+// const fun = async function () {
+//   const options = {
+//     method: "GET",
+//     headers: {
+//       accept: "application/json",
+//       Authorization:
+//         "Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiIzZmRkZDBhNmNjMzBhYWU2OWI4MzE0YTcyNWExNGQ0YSIsInN1YiI6IjY1MjdjNjA5MWYzZTYwMDExYzQ4ZTUxMSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.tpZsJkzTcCg9ZHaVuKmQFXNQ6vLPjoPkwfM_ggJoaVg",
+//     },
+//   };
+//   fetch(
+//     "https://api.themoviedb.org/3/movie/popular?language=en-US&page=1",
```

Comment:
This file has no code, please explain the need of this file.

Reply:
this was for api testing purpose. I am removing it